configure.ac

```diff
@@ -387,6 +394,16 @@ AS_VAR_IF([RKT_STAGE1_SYSTEMD_VER], [HEAD],
           [AC_MSG_WARN([* 'HEAD' is not a systemd version, setting it to 'master' instead])
            RKT_STAGE1_SYSTEMD_VER='master'])
 
+KVM_HV_TAG='hv_none'
+RKT_IF_HAS_FLAVOR([${RKT_STAGE1_FLAVORS}], [kvm],
+                  dnl for kvm check hypervisor
+                  AS_CASE([${RKT_STAGE1_KVM_HV}],
+                          [lkvm],
+                                  KVM_HV_TAG='hv_lkvm',
+                          [qemu],
```

Comment:
The `qemu` should be removed. It could be added back when qemu support actually lands in rkt.